### PR TITLE
adds in multi select for tagging

### PR DESCRIPTION
**Pull request by fredkingham**

### CODE REVIEW

**Comment by fredkingham** [comment]
fixes #1117

**Comment by fredkingham** [comment]
So we now have team as a sub field of episode and with a custom search rule.

We also have a new filter type, which gives the select option a many to many select. its only used in this case but I think its useful new search rules.

I've added a new filter for display arrays in a more human readable fashion. E.g ["apples", "pears"] gets translated to apples and pears.

I've removed all mention of 'tagging' in the template.

File: opal/core/search/search_rule.py [hunk 1]
```diff
@@ -109,7 +109,57 @@ def query(self, given_query):
             raise SearchException(err.format(given_query['queryType']))
 
 
+class EpisodeTeam(SearchRuleField):
+    ALL_OF = "All Of"
+    ANY_OF = "Any Of"
+
+    display_name = "Team"
+    description = "The team that looked after that episode of care"
```
Comment (davidmiller) [inline opal/core/search/search_rule.py]: I don't think that an episode of care is the subject of the verb in this sentence...

Comment (davidmiller) [inline opal/core/search/search_rule.py]: ...related, `the` is complicated b/c there can be more than one...

**Pull request merged**